pyvista/core/filters.py

```diff
@@ -1747,7 +1747,6 @@ def plot_over_line(dataset, pointa, pointb, resolution=None, scalars=None,
         if show:
             return plt.show()
 
-
```

Comment:
Just a personal note: I'd put such unrelated formatting into a separate commit and just either push that directly to master, as it's obviously harmless, or create a separate PR. It just adds pointless noise to the diff making a review just a bit less comfortable.

Reply:
True enough.  I'll avoid doing that in the future.